src/haddock/clis/cli_mpi.py

```diff
@@ -0,0 +1,70 @@
+"""Task running for libmpi."""
+import argparse
+import pickle
+import sys
+
+
+try:
+    from mpi4py import MPI
+except ImportError as e:
+    sys.exit(e)
+
+
+COMM = MPI.COMM_WORLD
+
+
+def split_tasks(task_l, n):
```

Comment:
No.

This script should be as self-contained as possible since its handling the MPI communications, its not advisable to add these cross references.

Reply:
Why? Isn't haddock3 installed when running the same as is python3 installed to import argparse? Don't see how the namespace will be contaminated making the script not functional. Also, or then, if this is supposed to be a stand-alone script to be executed independently from haddock3, it should be somewhere else. If you use the `main` function as part of some haddock3 internals and not as a command-line client, you should move the `main` to `libmpi` instead.

Reply:
Okay